### PR TITLE
Processor and handler for new nagios messages.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/nagios.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+# This file is part of fedmsg.
+# Copyright (C) 2013-2014 Red Hat, Inc.
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/nagios.py]: Should be 2015 no?

File: fedmsg_meta_fedora_infrastructure/nagios.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+class NagiosProcessor(BaseProcessor):
+    __name__ = "Nagios"
+    __description__ = "Fedora Infra Service Monitoring"
+    __link__ = "https://admin.fedoraproject.org/nagios"
+    __docs__ = "https://infrastructure.fedoraproject.org/infra/docs/nagios.rst"
+    __obj__ = "Service Outage Alerts"
+    __icon__ = "https://apps.fedoraproject.org/img/icons/nagios-logo.png"
+
+
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/nagios.py]: only 1 line?

File: fedmsg_meta_fedora_infrastructure/nagios.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    def subtitle(self, msg, **config):
+        states = {
+            'CRITICAL': self._('down'),
+            'WARNING': self._('having problems'),
+            'OK': self._('back up'),
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/nagios.py]: Do we publish Acknowledgments?

Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/nagios.py]: NM it's handled below

File: fedmsg_meta_fedora_infrastructure/tests/nagios.py [hunk 4]
```diff
@@ -0,0 +1,199 @@
+# This file is part of fedmsg.
+# Copyright (C) 2013 Red Hat, Inc.
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/nagios.py]: 2015?

File: fedmsg_meta_fedora_infrastructure/tests/nagios.py [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+        "msg": {
+            "host": "proxy04.fedoraproject.org",
+            "type": "PROBLEM",
+            "state": "CRITICAL",
+            "host_ack_author": "",
+            "service_ack_author": "",
+        }
+    }
+
+class TestLegacyNagiosServiceStateChange(Base):
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/tests/nagios.py]: Missing an empty line?

**Comment by pypingou** [comment]
A few stylish comment but nothing serious, :+1: for me, logic looks good

**Comment by ralphbean** [comment]
Thanks @pypingou!

**Pull request merged**